### PR TITLE
remove 'credentials' field from custom properties + rename 'name' as 'description' in OpenMetadata custom properties

### PR DESCRIPTION
**Pull request by cdoron**
Signed-off-by: Doron Chen <cdoron@il.ibm.com>

### CODE REVIEW

**Comment by cdoron** [comment]
address some of the points in https://github.com/fybrik/fybrik/issues/1800

**Comment by cdoron** [comment]
This PR removes `credentials` and `connectionType` from the OM custom properties.
It also sets a default value for dataFormat: csv
As a result, in some cases an OM user will not need to set the custom properties at all.

File: customization.yaml [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+  description: Parent Category for all generic labels
   tags:
   - name: PII
     description: Personally Identifiable Information
   - name: finance
     description: Financial Information
+- name: Purpose
+  description: The purpose for which assets may be used
+  tags:
+  - name: housing
```
Comment (roytman) [inline customization.yaml:14]: what is the housing tag ?
@simanadler probably uses it in the Kubeflow demo, but I don'y think we should add it by default.

File: pkg/database-types/database_type.go [hunk 2]
```diff
@@ -15,10 +15,11 @@ type DatabaseType interface {
 	OMTypeName() string
 
 	// translate the connection information from the Fybrik format to the OM format
-	TranslateFybrikConfigToOpenMetadataConfig(map[string]interface{}, *string) map[string]interface{}
+	TranslateFybrikConfigToOpenMetadataConfig(map[string]interface{}, string, *string) map[string]interface{}
```
Comment (roytman) [inline pkg/database-types/database_type.go:18]: there is no argument names, I'd suggest adding some explanation in the comments, what are the arguments. Currently, it's `string` and `*string`, what are they, why one is string and another *string ...

**Review by roytman — APPROVED** [review]

**Pull request merged**